ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Ledger/Query/Version.hs

```diff
@@ -24,3 +24,4 @@ nodeToClientVersionToQueryVersion x = case x of
   NodeToClientV_17 -> QueryVersion2
   NodeToClientV_18 -> QueryVersion2
   NodeToClientV_19 -> QueryVersion2
+  NodeToClientV_20 -> QueryVersion2
```

Comment:
I introduced a new query in #1373 that should make use of `NodeToClientV_20` too. Just FYI.